ui/src/app/pages/data/cohort-review/cohort-review.tsx

```diff
@@ -26,7 +26,7 @@ interface State {
   readonly: boolean;
 }
 
-export class CohortReview extends React.Component<{}, State> {
+export const CohortReview = withCurrentCohortCriteria()(class extends React.Component<{}, State> {
```

Comment:
this is just to use withCurrentCohortCriteria

Reply:
Ok, was asking because I don't think we'll need the current criteria here. Cohort review only uses cohorts that have been saved, so we shouldn't need access to any unsaved selections.